tests/e2e/utils/plugin-utils.js

```diff
@@ -1,98 +1,75 @@
-import { APIRequest } from '@playwright/test';
+import * as dotenv from 'dotenv';
 import axios from 'axios';
 import fs from 'fs';
 import path from 'path';
 
-/**
- * Encode basic auth username and password to be used in HTTP Authorization header.
- *
- * @param {string} username
- * @param {string} password
- * @returns Base64-encoded string
- */
-const encodeCredentials = ( username, password ) => {
-	return Buffer.from( `${ username }:${ password }` ).toString( 'base64' );
-};
+dotenv.config( {
+	path: `${ process.env.E2E_ROOT }/config/local.env`,
+} );
 
-/**
- * Deactivate and delete a plugin specified by the given `slug` using the WordPress API.
- *
- * @param {object} params
- * @param {APIRequest} params.request
- * @param {string} params.baseURL
- * @param {string} params.slug
- * @param {string} params.username
- * @param {string} params.password
- */
-export const deletePlugin = async ( {
-	request,
-	baseURL,
-	slug,
-	username,
-	password,
-} ) => {
-	// Check if plugin is installed by getting the list of installed plugins, and then finding the one whose `textdomain` property equals `slug`.
-	const apiContext = await request.newContext( {
-		baseURL,
-		extraHTTPHeaders: {
-			Authorization: `Basic ${ encodeCredentials( username, password ) }`,
-		},
-	} );
-	const listPluginsResponse = await apiContext.get(
-		`/wp-json/wp/v2/plugins`,
-		{
-			failOnStatusCode: true,
-		}
-	);
-	const pluginsList = await listPluginsResponse.json();
-	const pluginToDelete = pluginsList.find(
-		( { textdomain } ) => textdomain === slug
-	);
+const { GITHUB_TOKEN } = process.env;
 
-	// If installed, get its `plugin` value and use it to deactivate and delete it.
-	if ( pluginToDelete ) {
-		const { plugin } = pluginToDelete;
+const getReleaseInfo = async ( { repo, releaseTag } ) => {
+	const options = {
+		url: `https://api.github.com/repos/${ repo }/releases/${
+			releaseTag === 'latest' ? '' : 'tags/'
+		}${ releaseTag }`,
+		headers: {
+			Authorization: `token ${ GITHUB_TOKEN }`,
+		},
+	};
 
-		await apiContext.put( `/wp-json/wp/v2/plugins/${ plugin }`, {
-			data: { status: 'inactive' },
-			failOnStatusCode: true,
-		} );
+	// Make a request to the GitHub API to get information about the release
+	const releaseInfo = await axios( options ).catch( ( error ) => {
+		if ( error.response ) {
+			console.log( error.response.data );
+			console.log( error.response.status );
+			console.log( error.response.headers );
+		}
+		process.exit( 0 );
```

Comment:
Are these logs useful, or a leftover for debugging? I guess we could be a little more generic, like:

```JavaScript
const releaseInfo = await axios( options ).catch( ( error ) => {
	console.error( `GitHub API request failed: ${error.message}` );
	process.exit( 1 );
} );
```

I guess we should be exiting with `0` when the node script was run successfully, otherwise, we should use `1` or a greater number.